packages/desktop/src/renderer/index.tsx

```diff
@@ -37,16 +36,15 @@ const KeyDownHandler: React.FC<{ clearStore: () => void }> = ({ clearStore }) =>
 // Clears the persistent store but retains `Environment` to prevent misalignment
 // between the data source and the selected menu item in the app.
 const clearPersistentStore = () => {
-    const currentEnvironment = global.environment;
+    const currentEnvironment = global.environment || "PRODUCTION";
     persistentConfigService.clear();
     persistentConfigService.persist({ [PersistedConfigKeys.Environment]: currentEnvironment });
```

Comment:
like this seems like a conflict with the other persisted storage access - using one access method (persistentConfigService) seems ideal

Reply:
resolved in [94c1070](https://github.com/AllenInstitute/biofile-finder/pull/416/commits/94c1070949efacef943d7487829be2f23543fab0)